src/fuzzfetch/fetch.py

```diff
@@ -843,6 +839,15 @@ def from_args(cls, args=None, skip_dir_check=False):
         elif args.branch is None:
             args.branch = 'central'
 
+        if args.branch.startswith('esr'):
+            resp = _get_url('https://product-details.mozilla.org/1.0/firefox_versions.json')
+            key = 'FIREFOX_ESR' if args.branch is 'esr-stable' else 'FIREFOX_ESR_NEXT'
```

Comment:
`is` compares the object identity, not by value, so this shouldn't work as intended.